README.md

```diff
@@ -264,6 +264,8 @@ See sample configs in [configs](configs) folder or [examples](examples) for quic
   bf16: true # require >=ampere
   fp16: true
   tf32: true # require >=ampere
+  bfloat16: true # require >=ampere, use instead of bf16 when you don't want AMP
+  float16: true # use instead of fp16 when you don't want AMP
```

Comment:
Is there a gpu limitation for this? `float16`

Reply:
What does "when you don't want AMP" mean?

Reply:
@utensil , AMP stands for automatic mixed precision.

Reply:
I mean it's not mentioned elsewhere so might cause confusion, also the interaction of AMP and the float types chosen here is not so clear.

Reply:
I agree the naming is a bit unclear.

Reply:
I added some additional copy, hopefully it helps